core/Cargo.toml

```diff
@@ -76,7 +76,8 @@ tower = "0.5.2"
 jemallocator = { version = "0.6.0", package = "tikv-jemallocator", optional = true }
 jemalloc-ctl = { version = "0.6.0", package = "tikv-jemalloc-ctl", optional = true }
 foundry-compilers = "0.18.0"
-alloy-chains = "0.2.4"
+alloy-chains = "0.2.2"
```

Comment:
did we downgrade for a reason?

Reply:
probably my oversight during conflict resolution